scripts/exregional_run_analysis.sh

```diff
@@ -674,6 +674,7 @@ if [[ ${gsi_type} == "ANALYSIS" && ${anav_type} == "radardbz" ]]; then
 fi
 if [[ ${gsi_type} == "ANALYSIS" && ${anav_type} == "conv_dbz" ]]; then
   ANAVINFO=${FIX_GSI}/${ANAVINFO_CONV_DBZ_FN}
+  beta1_inv=0.0
```

Comment:
@hu5970 @chunhuazhou Does it mean that all previous experiments used only 85% ensemble increments for ref DA?

Reply:
@guoqing-noaa @hu5970 @chunhuazhou This part is effective only in the case of ${anav_type}=="conv_dbz" (simultaneous dbz DA). I deleted this "beta1_inv=0.0" to make simultaneous dbz DA possible with hybrid-EnVar in [PR#533](https://github.com/NOAA-GSL/regional_workflow/pull/533).

I don't know why "beta1_inv=0.0" was reverted here in this PR. I think "beta1_inv=0.0" shouldn't be added here.

Reply:
I will delete "beta1_inv=0.0"  in next PR.